packages/react-menu/tsconfig.json

```diff
@@ -1,23 +1,16 @@
 {
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "baseUrl": ".",
+    "target": "ES5",
+    "lib": ["ES5", "dom"],
     "outDir": "dist",
-    "target": "es5",
-    "module": "commonjs",
     "jsx": "react",
     "declaration": true,
-    "sourceMap": true,
     "experimentalDecorators": true,
     "importHelpers": true,
     "noUnusedLocals": true,
-    "forceConsistentCasingInFileNames": true,
-    "strict": true,
-    "moduleResolution": "node",
     "preserveConstEnums": true,
```

Comment:
This should also be in the defaults. Not preserving const enums can cause problems for consumers in some cases.

Reply:
Possibly a valid point, but it's not something we've discussed specifically in this repo (aside from that we shouldn't use const enums due to compat issues with other build systems). If we want to ban enums, that should be done with a lint rule. Until then, we should have defaults that reasonably handle the last agreed upon state of things, which currently is that enums are allowed. `preserveConstEnums` is probably less important for tooling packages but not significantly harmful either.

Reply:
I'll keep this config scoped per package for reasons I mentioned for now.

> that should be done with a lint rule
cant agree more. we can/should iterate on linting approach etc in a follow up PR/RFC